tests/test_user_modules.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import os
+
+import pkg_resources
+import pytest
+
+import py3status
+from py3status.core import Py3statusWrapper, ENTRY_POINT_KEY
+
+
+@pytest.fixture(name="status_wrapper")
+def make_status_wrapper():
+    args = argparse.Namespace()
+    status_wrapper = Py3statusWrapper(args)
+    return status_wrapper
+
+
+def test__get_path_based_modules(status_wrapper):
+    """Use the list of inbuilt modules as reference to check against."""
+    included_modules_path = os.path.join(os.path.dirname(py3status.__file__), "modules")
+    status_wrapper.options.__dict__["include_paths"] = [included_modules_path]
+    assert os.path.exists(included_modules_path)
+    expected_keys = [
+        n[:-3] for n in os.listdir(included_modules_path) if n.endswith(".py")
+    ]
+    modules = status_wrapper._get_path_based_modules()
+    assert sorted(modules.keys()) == sorted(expected_keys)
+
+
+def test__get_entry_point_based_modules(status_wrapper, monkeypatch):
+    def return_fake_entry_points(*_):
+        class FakePy3status(object):
+            Py3status = "I am a fake class"
+
+            def __init__(self, name):
+                self.name = name
+                self.module_name = "module_name_" + self.name
+
+            @staticmethod
+            def load():
```

Comment:
not sure why on python3.6 this breaks travis : https://travis-ci.org/ultrabug/py3status/jobs/558894076

Reply:
ah - linting, should be fixed with last commit